app/kumactl/cmd/install/render_helm_files_gateway.go

```diff
@@ -0,0 +1,17 @@
+//go:build gateway
+// +build gateway
+
+package install
+
+import (
+	"github.com/kumahq/kuma/app/kumactl/pkg/install/data"
+)
+
+func init() {
+	gatewayEnabled = true
+	helmTemplateFilters = []fileFilter{
+		func(f data.File) bool {
+			return true
+		},
+	}
+}
```

Comment:
You don't need this file at all. If `helmTemplateFilter` is an empty slice, all the files will be accepted.

Reply:
🤦